cmake/CMakeLists.txt

```diff
@@ -1173,6 +1173,16 @@ if (onnxruntime_ENABLE_TRAINING)
           message(STATUS "NCCL_MINOR_VERSION: ${NCCL_MINOR_VERSION}")
         endif()
 
+        if (${NCCL_MAJOR_VERSION} GREATER 2)
```

Comment:
nit: maybe this condition can be simplified with VERSION_GREATER_EQUAL
https://cmake.org/cmake/help/v3.14/command/if.html#condition-syntax

Reply:
Sure. Here is a PR https://github.com/microsoft/onnxruntime/pull/5118/files.